pkg/sql/backfill.go

```diff
@@ -141,17 +141,12 @@ func (sc *SchemaChanger) fixedTimestampTxn(
 // This operates over multiple goroutines concurrently and is thus not
 // able to reuse the original client.Txn safely. The various
 // function that it calls make their own txns.
-func (sc *SchemaChanger) runBackfill(
-	ctx context.Context, lease *sqlbase.TableDescriptor_SchemaChangeLease,
-) error {
+func (sc *SchemaChanger) runBackfill(ctx context.Context) error {
 	if sc.testingKnobs.RunBeforeBackfill != nil {
 		if err := sc.testingKnobs.RunBeforeBackfill(); err != nil {
 			return err
 		}
 	}
-	if err := sc.ExtendLease(ctx, lease); err != nil {
```

Comment:
2¢: I think it might be good to include checks to `ctx.Err()` where we used to check for lease loss, since job-lease-loss would be communicated that way. We'll _probably_ find it anyway when we go to do an RPC but it'd be nice to have some explicit checks at least as a often as we used to check the lease i think?